### PR TITLE
Beautify Readme.md and update feature section

### PR DESCRIPTION
**Pull request by JohannesLandgraf**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,16 +17,21 @@ Tightly integrated with GitLab, GitHub, and Bitbucket, Gitpod automatically and
...
- - Integrated Docker build
- - GitLab, GitHub, and Bitbucket integration
- - Integrated code reviews
- - Snapshots - Snapshot any state of your dev environment and let others create clones
- - Collaboration - Invite team members to your environments
- - Gitpod CLI to automate your experience
- - [learn more](https://www.gitpod.io/features/)
+🏗 [Dev environments as code](https://www.gitpod.io/docs/#-dev-environments-as-code) - Gitpod applies lessons learned from infrastructure-as-code. Spinning up dev environments is easily repeatable and reproducible empowering you to automate, version-control and share dev environments across your team.
+
+⚡️ [Prebuilt dev environments](https://www.gitpod.io/docs/#prebuilds) - Gitpod continuously builds all your git branches like a CI server. Whenever new commits are pushed to your repository, Gitpod can prebuild dev environments, i.e. run the init commands in your `.gitpod.yml` before you even start a workspace.
```
Comment (svenefftinge) [inline README.md:22]: Gitpod continuously prebuilds dev environments for all you git branches similar to a CI server. Control how Gitpod prepares environments through `init` commands in your `.gitpod.yml`.

Comment (svenefftinge) [reply README.md:22]: i.e. Gitpod doesn't build your branches.

File: README.md [hunk 2]
```diff
@@ -11,16 +17,21 @@ Tightly integrated with GitLab, GitHub, and Bitbucket, Gitpod automatically and
...
+
+🐳 [Integrated Docker build](https://www.gitpod.io/docs/config-docker/) - Gitpod instantly starts a container in the cloud based on your Docker image. Tools that are required for your project are easy to install and configure.
+
+👐 [GitLab, GitHub, and Bitbucket integration](https://www.gitpod.io/docs/integrations/) - Gitpod seamlessly integrates in your workflow and works with all major git hosting platforms including GitHub, GitLab and Bitbucket.
+ 
+👀 [Integrated code reviews](https://www.gitpod.io/docs/code-reviews/#code-reviews) - with Gitpoid you can do native code reviews on any PR/MR. No need to switch context anymore and declutter your local machine with your colleagues PR/MR.
```
Comment (svenefftinge) [inline README.md:28]: typo: Gitpod

second sentence: declutter -> clutter

File: README.md [hunk 3]
```diff
@@ -11,16 +17,21 @@ Tightly integrated with GitLab, GitHub, and Bitbucket, Gitpod automatically and
...
+
+👯‍♀️ [Collaboration](https://www.gitpod.io/docs/sharing-and-collaboration/) - invite team members to your dev environment or snapshot any state of your dev environment to share it with your team.
```
Comment (svenefftinge) [inline README.md:30]: ... dev environment to share it with your team **asynchronously**.

File: README.md [hunk 4]
```diff
@@ -1,4 +1,10 @@
-<img alt="Gitpod Logo" src="https://raw.githubusercontent.com/gitpod-io/gitpod/master/components/dashboard/public/images/gitpod-ddd_light.svg" width="300">
+<p align="center">
+  <a href="https://www.gitpod.io">
+    <img src="https://raw.githubusercontent.com/gitpod-io/gitpod/master/components/dashboard/public/images/gitpod.svg" height="60">
+    <h3 align="center">Gitpod</h3>
+  </a>
+  <p align="center">Always ready-to-code.</p>
+</p>
```
Comment (svenefftinge) [inline README.md:7]: No strong opinion here, but I like the existing left alignment better. It somehow is easier on the eye/reading flow.

**Review by svenefftinge — APPROVED** [review]
+1 just some minor feedback

File: README.md (hunk 4 above)
Comment (JohannesLandgraf) [reply README.md:7]: Got inspired by https://github.com/vercel/vercel and think the centralized structure looks cleaner and gives us the option to display both 1) link to the website and 2) our claim.

**Pull request merged**